microsoft-edge/devtools/workspaces/workspace-tutorial.md

```diff
@@ -87,20 +87,20 @@ Continue with the next section.
 
 1. Create a working branch named "test" and switch to it (to avoid altering the demo's files in the "main" branch):
 
-   ```Shell
+   ```shell
    git checkout -b test
    ```
 
    That's equivalent to:
 
-   ```Shell
+   ```shell
    git branch test
    git switch test
    ```
 
 1. Run one of the following commands, to start the web server:
 
-   ```bash
+   ```shell
```

Comment:
this is the only spot maybe changing from bash (to shell), but I might revert even this change, b/c:
* good to do b/c the above langs are shell.
* bad to do b/c the below langs, grouped w this block, are specifically bash, w/ ~ and /
* this code lacks ~ or / so should be ok regardless of whether user uses git bash vs. Win Cmd Prompt.

@leahmsft